public/docs/ts/latest/guide/architecture.jade

```diff
@@ -193,37 +186,32 @@ figure
 figure
   img(src="/resources/images/devguide/architecture/template.png" alt="Template" align="left" style="width:200px; margin-left:-40px;margin-right:10px" )
 :marked
-  We define a component's view with its companion **template**. A template is a form of HTML
+  You define a component's view with its companion **template**. A template is a form of HTML
   that tells Angular how to render the component.
 
-  A template looks like regular HTML much of the time ... and then it gets a bit strange. Here is a
+  A template looks like regular HTML much of the time but with some differences. Here is a
   template for our `HeroListComponent`:
 
 +makeExample('app/hero-list.component.html')
 
 :marked
   This template features typical HTML elements like `<h2>` and  `<p>`.
-  But what are `*ngFor`, `{{hero.name}}`, `(click)`, `[hero]`, and `<hero-detail>`?
+  It also includes `*ngFor`, `{{hero.name}}`, `(click)`, `[hero]`, and `<hero-detail>` which are examples of Angular's [template syntax](template-syntax.html).
 
-  These are examples of Angular's [template syntax](template-syntax.html).
-  We will grow accustomed to that syntax and may even learn to love it.
-  We'll begin to explain it in a moment.
 
-  Before we do, focus attention on the last line.
+  Focus attention on the last line.
```

Comment:
I'm not crazy about this transition. How about.

In the last line of the template, the `<hero-detail>` tag is a custom element that represents a new component, `HeroDetailComponent`.